sql/12/cwvs-by-wordpress-version.sql

```diff
@@ -0,0 +1,92 @@
+# HTTP Archive query to get Core Web Vital "good" rates by WordPress version.
```

Comment:
This should be in a `2022/12` directory, not just `12`.

Reply:
moved to `2023/01`